src/Compilers/VisualBasic/Portable/LanguageFeatures/LanguageFeature.vb

```diff
@@ -39,6 +39,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
     End Enum
 
     Friend Module FeatureExtensions
+        <Extension>
+        Friend Function GetFeatureFlag(feature As Feature) As String
+            Select Case feature
+                'Case feature.IOperation
+                '    Return "IOperation"
+
+                Case Else
+                    Return Nothing
+            End Select
```

Comment:
This method is incomplete

Reply:
Are you sure? From my understand it returns `Nothing` for all cases, when the code is commented out. 
Which is do to `IOperation` not being in the definitive feature list.

Reply:
What method is this replacing? The only use of IOperation is in the new code here and it is commented out.

Reply:
It is for features that are enabled via the cmd line feature flag.